bucket/rakudo-star.json

```
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://rakudo.org/",
+    "version": "2020.01-01",
+    "license": "Artistic-2.0-Perl",
+    "description": "The most mature, production-ready implementation of the Raku language",
```

- `Artistic-2.0-perl` license do not exists

```suggestion
    "version": "2020.01-01",
    "description": "Raku compiler for the MoarVM and JVM",
    "homepage": "https://rakudo.org",
    "license": "Artistic-2.0",
```

---

Thanks. Copied it from the PERL.json (if I'm not wrong)...